docs/DeveloperGuide.adoc

```diff
@@ -508,6 +513,19 @@ The result is then displayed into the GUI for the user as the cost price of the
 [NOTE]
 It is possible for the LoanBook to have a loan that lasts for 0 minutes. This is because there may be some weird edge case where an object is loaned for less than a minute, which gets prorated down.
 
+[big red]#Sequence diagram#
+
+The sequence diagram has been split into 2 parts. +
+First, the sequence diagram of command parsing:
+
+image::ReturnSequence.png[width="600"]
+
+{nbsp} +
+
+Second, the sequence diagram of command execution:
+
+image::ReturnSequenceLogic.png[width="600"]
```

Comment:
For the "AIOOB" and "LoanStatus not ONGOING" alt blocks, you could label the outgoing arrows with the exception thrown as well.